src/PhpCsFixer/PhpCsFixerLicense.php

```diff
@@ -25,7 +25,7 @@
 
 class PhpCsFixerLicense
 {
-    private const YEAR = 2023;
+    private const YEAR = 2024;
```

Comment:
No need to change for next year.
```suggestion
    private const YEAR = (new DateTime())->format('Y');
```